### DISCUSSION TITLE
Не работает быстрая форма.

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#497] [ОШБК!] [answered]
<div id="form" style="width: 100%; min-height: 200px; max-width: 620px; margin: auto">
<script>
let num = "22";
let host = "https://armsocc.ru/";
let formName = "q_form_reg";
let div = document.currentScript.parentNode;
let newScript = document.createElement("script");
newScript.charset = "utf-8";
newScript.onerror = (event) => {
    console.log(event)
    div.innerHTML = '<div>It didn\'t load.:(</div>';
};
newScript.onload = (event) => {
    ttmForm(div, host + "Forms/" + formName, true, [], window.location.search, "");
};
div.insertBefore(newScript, document.currentScript);
newScript.src = host + "/js/forms.js?" + num;
if (!window.ttmCss) {
    window.ttmCss = true;
    let styles = document.createElement("link");
    styles.rel = "stylesheet";
    styles.type = "text/css";
    styles.href = host + "/css/forms.css?" + num;
    document.head.append(styles);
}
</script>
</div>

Не открывается эта быстрая форма. Что не так в настройках?
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/114401808/bce8d111-99ba-4038-af17-c350d78344d3)

Экран выглядит так:

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/114401808/7a6e4411-61f3-4489-8758-89db8280ad4d)

**Comment by ttm-alexeypolunin** [comment]
Проверьте, что у вас все поля перечисленные в настройках формы существуют в таблице.

**Reply by vldekov** [reply]
Делал по инструкции. Все поля взяты автоматически из таблицы. Поля все есть в таблице.

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/114401808/815ab413-7988-4662-8fbb-0c887483eca3)
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/114401808/5c682bac-4c68-46c0-8b4e-e2bd4367820d)

**Reply by ttm-alexeypolunin** [reply]
4.11.57 актуальная версия. Обновите плиз.

**Comment by vldekov** [comment]
Быстрые формы не работают и с другими таблицами в других проектах. Что то общая вроде проблема.

**Reply by Re-Requeste** [reply]
> Быстрые формы не работают и с другими таблицами в других проектах. Что то общая вроде проблема.
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/17f83f20-c765-4052-a49e-282b9e1d88dd)

Проверил на 4.10.56 - работает хорошо, обновился на 4.11.57 - работает хорошо

**Comment by vldekov** [comment, marked as answer]
Обновился до версии 4.11.57. Быстрые формы заработали. Спасибо!
